cmd/odf/set/backfillfull_ratio.go

```diff
@@ -0,0 +1,28 @@
+package set
+
+import (
+	"fmt"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+var osdBackFillFullRatioCmd = &cobra.Command{
+	Use:                "backfillfull",
+	Short:              "Configure ceph osd 'backfillfull-ratio' setting",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+	Example:            "odf set ceph backfillfull-ratio 0.9",
```

Comment:
For all the commands:
```suggestion
	Example:            "odf set backfillfull-ratio 0.9",
```

Reply:
done